scripts/prepare.sh

```diff
@@ -28,30 +28,6 @@ else
   CDIR=courses
 fi
 
-# Parse arguments.
```

Comment:
Why is this file `scripts/prepare.sh` not completely deleted? It is only used in the chroot sandbox, right?

Reply:
Done!